package.json

```diff
@@ -6,13 +6,20 @@
   "scripts": {
     "test": "bazel test //..."
   },
-  "dependencies": {
-    "@bazel/worker": "^4.2.0"
+  "bin": {
+    "webpack-worker": "./webpack/worker/worker_adapter.js"
   },
-  "peerDependencies": {
+  "dependencies": {
+    "@bazel/worker": "^4.2.0",
     "webpack-cli": ">=4",
     "webpack": ">=5"
   },
+  "devDependencies": {
+    "@bazel/bazelisk": "^1.10.1",
+    "@bazel/typescript": "^4.2.0",
+    "@types/node": "^16.9.4",
+    "typescript": "^4.3.3"
```

Comment:
I don't think we're doing enough typechecking for this to be worth it.

If we author in plain JS we can just ship our sources instead of needing a build system

Reply:
I wrote it in ts just to have code intellisense.  I’d prefer to use the JSDoc equivalent more